docs/CONTRIBUTING.md

```diff
@@ -111,6 +111,7 @@ If you are a Reviewer or below, then once you have approved a PR, request a revi
 #### Triaging Bugs
 
 New bugs need to be triaged to identify the highest priority ones.
+Any members can perform triaging.
```

Comment:
```suggestion
Any Member can triage bugs.
```

- capitalize "Member" as other roles are capitalized in this doc
- use consistent verbiage, "triage bugs", rather than "perform triaging".
  - this is [simpler](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language) and closer to [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense) / a simpler tense (literally future simple tense vs. future continuous -- yes I had to look that up as I don't remember all the variations of tenses 😅)